bugsnag-plugin-android-ndk/src/main/assets/include/bugsnag.h

```diff
@@ -36,6 +39,12 @@ void bugsnag_set_user_env(JNIEnv *env, char* id, char* email, char* name);
 void bugsnag_leave_breadcrumb(char *message, bsg_breadcrumb_t type);
 void bugsnag_leave_breadcrumb_env(JNIEnv *env, char *message, bsg_breadcrumb_t type);
 
+void bugsnag_add_on_error(bsg_on_error on_error);
```

Comment:
The new functions in this file need some documentation in particular, as this is the public header.